clientd3d/d3drender.c

```diff
@@ -6979,7 +6982,17 @@ void D3DRenderObjectsDraw(d3d_render_pool_new *pPool, room_type *room,
 		pChunk->numPrimitives = pChunk->numVertices - 2;
 		pChunk->xLat0 = xLat0;
 		pChunk->xLat1 = xLat1;
-		pChunk->zBias = ZBIAS_BASE;

```

Comment:
A clearer way to write this is

```
pChunk->zBias = ZBIAS_BASE;
// Nodes with a bound height adjust are part of other players' upper bodies.
if (pRNode->boundingHeightAdjust == 0)
{
	// Typical items such as reagents, keys, etc.
	pChunk->zBias = ZBIAS_DEFAULT+(z_depth_inc++);
}
```

Is this always totally safe?  As in, no "typical items" are allowed to have overlays?